packages/d/dotnet/package.yml

```diff
@@ -0,0 +1,83 @@
+name       : dotnet
+version    : 8.0.1
+release    : 1
+source     :
+    - git|https://github.com/dotnet/dotnet.git : b27976e5a6850466ee5b4ce24f91ee93bef645f7
+homepage   : https://dot.net
+license    : MIT
+component  : programming.dotnet
+summary    :
+    - .NET runtime
+    - sdk : .NET Development Kit
+    - source-built-artifacts : Internal package for building .NET
+description: .NET is a free, cross-platform, open-source developer platform for building many kinds of applications. It can run programs written in multiple languages, with C# being the most popular.
+replaces   :
+    - fsharp
+networking : yes
+libsplit   : no
+clang      : yes
+strip      : no
+builddeps  :
+    - pkgconfig(icu-i18n)
+    - pkgconfig(krb5)
+    - pkgconfig(libcurl)
+    - pkgconfig(libssl)
+    - pkgconfig(libunwind)
+    - pkgconfig(libxml-2.0)
+    - pkgconfig(lttng-ust)
+    - pkgconfig(zlib)
+    # - dotnet-sdk
+    # - dotnet-source-built-artifacts
+    - git
+    - lldb-devel
+rundeps    :
+    - libicu
+    - sdk:
+        - dotnet
```

Comment:
The auto-dep might actually have already figured this out. You should take this part out and try building it again. Open up the -sdk file in a archive viewer and look at the metadata. I imagine it's already in there.